Sources/TuistGenerator/Generator/ProjectFileElements.swift

```diff
@@ -501,6 +496,10 @@ class ProjectFileElements {
         path.extension == "lproj"
     }
 
+    func isPlayground(path: AbsolutePath) -> Bool {
+        path.extension == "playground"
+    }
+
```

Comment:
Does this mean that it will not recurse into this folder and gobble up the *.swift files it contains in there?